tests/specs/run/op_exit_op_set_exit_code_in_worker/op_exit_op_set_exit_code_in_worker.ts

```diff
@@ -2,8 +2,7 @@
 // setting exit code in worker context is a no-op and is an alias for
 // `self.close()`.
 
-// @ts-ignore Deno[Deno.internal].core doesn't have type-defs
-Deno[Deno.internal].core.ops.op_set_exit_code(21);
+Deno.exitCode = 21;
```

Comment:
See

https://github.com/denoland/deno/blob/d17ae8ca9b0777c64934ae332b83ff0996a9fae0/runtime/js/99_main.js#L594-L596

and 

https://github.com/denoland/deno/blob/d17ae8ca9b0777c64934ae332b83ff0996a9fae0/ext/os/30_os.js#L105-L117